lua/neo-tree/ui/renderer.lua

```diff
@@ -356,7 +356,7 @@ local enable_auto_close_floats = function()
   auto_close_floats_is_set = true
 end
 
-local create_window = function(state)
+create_window = function(state)
```

Comment:
Why remove the local here?

Reply:
It is already declared as a `local` variable at the top of the file.